articles/tales-from-fleet-security-securing-bank-accounts-from-business-email-compromise.md

```diff
@@ -0,0 +1,112 @@
+
+If possible, enable at least the following:
+
+1. Alert when a new recipient is added
+2. Alert when a wire transfer is created (domestic or international)
+3. Alert when an external transfer is created
+4. Alert when an address or other contact information is changed
+5. Alert on any transaction above a certain threshold, such as $10,000
+
+These alerts are typically delivered via email, which is not ideal for rapid awareness. To improve that, create an email address never used anywhere else specifically for these alerts. You'll then be able to integrate that mailbox to other tools, like Slack, by using a [forwarding address](https://slack.com/help/articles/206819278-Send-emails-to-Slack#:~:text=To%20send%20an%20email%20to,see%20your%20email%20in%20Slack.). 
+With this configuration, you can deliver these alerts in your administrative or accounting channel, where everyone can see them, create a thread and ask questions.
+
+On top of email alerts, inform your accounting department that they should pay attention to push notifications from your online banking application. If you can keep the amount of "false positives" down and your accounting and finance are all in the same time zone, have a rotation for who will let the app notify them even when in do not disturb mode.
+
+### Dual and triple approval
+
+Most business banks allow customers to set thresholds above which the system will require multiple approvals.
+A threshold of $10,000 before a second approval is needed ensures that if an attacker compromised a single person, they could only send away money in $10,000 batches. Alerting remains essential as the total cost can grow pretty quickly, but it is much safer than allowing a single person to move millions of dollars.
+
+Consider requiring three approvals for substantial transactions, as long as your team is large enough to accomodate this even when people are on vacation or unavailable for any other reason.
+
+### ACH filters
+
+ACH stands for Automated Clearing House, a network used for direct payments, especially in the US. If your company is in the US and deals with US vendors, you probably send most of your payments with ACH.
+
+Many banks offer ACH filters on business accounts. These filters work like many security controls, using one of two main methods:
+
+* In deny list mode, where you provide criteria to deny payments
+* In allow list mode, where you give criteria to allow payments
+
+Allow listing is safer but requires more work to implement. Some banks allow combinations of vendors and amounts while others only allow lists of vendors. 
+
+Before implementing ACH filters, obtain a list of all ACH transfers made in the last 18 months. Identify the vendors you have that are still active, the average amount you pay them, plus a buffer to cover increases, and then provide this list to your bank to enable ACH filtering in allow list mode. Have every other payment generate an **exception** that will require manual intervention.
```

Comment:
```suggestion
Before implementing ACH filters, obtain a list of all ACH transfers made in the last 18 months. Identify the vendors you have that are still active, the average amount you pay them, plus a buffer to cover increases, and then provide this list to your bank to enable ACH filtering to allow list mode. Have every other payment generate an **exception** that will require manual intervention.
```

Reply:
@GuillaumeRoss, should this suggestion be dismissed or committed?

Reply:
Dismissed!